strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/layout/maven/MavenMetadataManagementControllerTest.java

```diff
@@ -144,41 +106,43 @@ public void testRebuildSnapshotMetadata()
                .then()
                .statusCode(HttpStatus.OK.value());
 
+        //artifactMetadataService.
```

Comment:
Cosmetic: Could you please remove this commented out line?

Reply:
Done.